indico/modules/events/editing/client/js/components/FileTypeManager.jsx

```diff
@@ -41,6 +42,18 @@ function fileTypesReducer(state, action) {
   }
 }
 
+function statusIcon(fileParam, iconName, text) {
+  return (
+    <Icon
+      styleName={fileParam ? '' : 'disabled'}
+      size="small"
+      color={fileParam ? 'blue' : 'grey'}
+      name={iconName}
+      title={fileParam ? Translate.string(text) : ''}
```

Comment:
Don't pass dynamic values to `Translate.string()`; this doesn't work since there's no way those values can be extracted for translation. Pass `Translate.string('...')` when using the StatusIcon component instead.